getlantern.org/static/js/main.js

```diff
@@ -1,57 +1,20 @@
 $(document).ready(function(){
 
-  var ANDROID_LINK = "lantern-installer-beta.apk";
```

Comment:
This, and below download related functions, are moved to `download.js`, to be reused in both `index.html` and `outdated.html`